web_src/css/modules/tippy.css

```diff
@@ -56,6 +61,15 @@
   fill: var(--color-menu);
 }
 
+.tippy-box[data-theme="menu"] .tippy-content > .ui.menu {
+  width: auto; /* unset fomantic's default 15rem */
+  border: none; /* remove `.ui.menu` border as it is provided by tippy */
+}
+
+.tippy-box[data-theme="menu"] .tippy-content .ui.vertical.menu .item::before {
+  content: none; /* remove lines between items */
+}
```

Comment:
Is it possible to get rid of the Fomantic defined classes? I do not think it is maintainable to mix different frameworks together.

Reply:
Likely, but then I need to come up with a few new rules to style menus in tippy. But good idea, I will check. This will affect also the "line button" in code view.

Reply:
I think I may re-purpose the `.item`class, as I assume fomantic would not touch it if it's standalone with no `.menu` parent.

Reply:
Done, removed the dependency from both menus:

<img width="201" alt="image" src="https://github.com/go-gitea/gitea/assets/115237/0c946955-11d9-45e2-aa67-c956c8efed03">

<img width="367" alt="image" src="https://github.com/go-gitea/gitea/assets/115237/7ae81513-a465-4e16-8c44-8dd07cfcf98d">